packages/fx-core/resource/package.nls.json

```diff
@@ -869,6 +869,7 @@
   "driver.oauth.log.skipCreateOauth": "Environment variable %s exists. Skip creating API key.",
   "driver.oauth.log.oauthNotFound": "Environment variable %s exists but unable to retrieve OAuth registration from Developer Portal. Check manually if it exists.",
   "driver.oauth.error.nameTooLong": "The OAuth name is too long. The maximum character length is 128.",
+  "driver.oauth.error.oauthDisablePKCEError": "Disabling PKCE for OAuth2 is not supported in the oauth/update action.",
```

Comment:
Revised: Turning off PKCE for OAuth2 is not supported in the oauth/update action.
Logic: As per our style guide, terms 'enable or disable' are banned.